docs/knowledgebase/getting-started/index.md

```diff
@@ -2,67 +2,104 @@
 title: Installation
 ---
 
-## Fast Installation
+This page will guide you through the steps needed to prepare a computer for Substrate development.
+Since Substrate is built with [the Rust programming language](https://www.rust-lang.org/), the first
+thing you will need to do is prepare the computer for Rust development - these steps will vary based
+on the computer's operating system. Then, you can use the helpful utilities from the Rust toolchain
+to configure the Rust development environment - these steps will be the same for all the
+(Unix-based) operating systems discussed on this page.
 
-Install all the required dependencies with a single command. (Be patient, this can take up to 30
-minutes)
+## Operating System-Dependent Set-Up
+
+Follow the steps for your computer's operating system before installing and configuring the Rust
+toolchain.
+
+### Unix-Based Operating Systems
+
+Substrate development is easiest on Unix-based operating systems, like macOS or Linux. The examples
+in the Substrate [Tutorials](../../../../tutorials) and [Recipes](https://substrate.dev/recipes/)
+use Unix-style terminals to demonstrate how to interact with Substrate from the command line.
+
+#### macOS
+
+Open the Terminal application and execute the following commands:
 
 ```bash
-curl https://getsubstrate.io -sSf | bash -s -- --fast
-```
+# Install Homebrew if necessary https://brew.sh/
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 
-## Manual Installation
+# Make sure Homebrew is up-to-date and install some dependencies
+brew update
+brew install openssl cmake llvm
+```
 
-### Debian
+#### Ubuntu/Debian
 
-Run:
+Use your favorite terminal shell to execute the following commands:
 
 ```bash
-sudo apt install -y cmake pkg-config libssl-dev git gcc build-essential clang libclang-dev
+sudo apt update
+# May prompt for location information
```

Comment:
I don't think it's when you run `apt` I think it _may_ be installing the SSL library although I'm honestly not sure. It prompts for physical, geographical location information, such as timezone. The reason I think we should mention it is because the person may step away from their computer and come back 30-minutes later to a screen that is waiting for them to respond to a prompt. I think that may have happened to "my friend" or something :sweat_smile: